ui/src/routes/Home/components/HomeView.js

```diff
@@ -108,61 +44,77 @@ class HomeView extends Component {
   }
 
   renderUserTypeButtons = (label, index) => {
-    let className = "user-select"
+    let className = 'user-select'
+    let radioDot = 'circle-o'
+    let ariaHid = false
     let checked = false
     if (this.props.survey && this.props.survey.userType === label) {
-      className = "user-select button-gradient-active"
+      className = 'user-select button-gradient-active'
+      radioDot = 'dot-circle-o'
+      ariaHid = true
       checked = true
     }
     return (
-      <div className={className}>
-        <label key={index}
-          htmlFor={label}>
-          <input key={index}
-            id={label}
-            onChange={(e) => this._onHandleUserTypeSelect(e)}
-            type="radio"
-            name="userType"
-            value={label}
-            checked={checked}
-            className="user-select__input"
-            ref={(input) => { this.inputField = input; }}
-            />
-          {label}
-        </label>
-      </div>
+      <label htmlFor={label} className={className}>
```

Comment:
Do you get a react warning about this not having a `key` attribute? I think you might want to set `key={index}` here on the `<label>`...

Reply:
I'm not getting a React warning about this not having a key attribute. 

Again, this PR is about CSS cleanup, not React components or HTML structure.

The React/HTML structure will be addressed in the accessibility feature branch I'm beginning after this PR is approved. :)